### DISCUSSION TITLE
Plugin is misconfigured and cannot be activated : Saleor 3.0 beta admin emails plugin

### DISCUSSION THREAD

**Discussion by Batgerel53** [saleor/saleor#8839] [Q&A]
![ss](https://user-images.githubusercontent.com/56902009/146754459-d928bbef-4b71-4272-ba5c-543e2129fb9f.PNG)
![sdf](https://user-images.githubusercontent.com/56902009/146754479-61ca608f-148d-4f1a-a4c5-f1e7291f6bec.PNG)

**Comment by karolkielecki** [comment]
@Batgerel53 I've passed it to the team. It's a busy time for us. We'll try to get back to you as soon as we can.

**Reply by Batgerel53** [reply]
Thank you @karolkielecki  . I solved this problem :)

**Reply by zamaniramin549** [reply]
May I know how did you solve this issue?

**Comment by zamaniramin549** [comment]
was this issue fix? I have the same issue
